features/networking/operator.feature

```diff
@@ -228,14 +228,14 @@ Feature: Operator related networking scenarios
   And the output should contain:
     | The service "<%= project.name %>/test-service" has been marked as idled |
   When I execute on the "hello-pod" pod:
-    | /usr/bin/curl | --connect-timeout | 10 | <%= cb.service_ip %>:27017 |
+    | /usr/bin/curl | --connect-timeout | 30 | <%= cb.service_ip %>:27017 |
   Then the step should fail
   #Moving CNO config back to normal and expect service to unidle then
   Given as admin I successfully merge patch resource "networks.operator.openshift.io/cluster" with: 
     | {"spec":{"defaultNetwork":{"openshiftSDNConfig": null}}} |
   And 60 seconds have passed
   When I execute on the "hello-pod" pod:
-    | /usr/bin/curl | --connect-timeout | 10 | <%= cb.service_ip %>:27017 |
+    | /usr/bin/curl | --connect-timeout | 30 | <%= cb.service_ip %>:27017 |
```

Comment:
Apparently it might take some time to idle the svc so 30 seconds seems okay to me. tested multiple times